gradle/spotbugs.gradle

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+
+
+plugins.withId('com.github.spotbugs') {
+    spotbugs {
+        toolVersion = spotbugsVersion
+        effort = "default"        
+        includeFilter = file("$rootDir/checkstyle/spotbugs-include.xml")
+        excludeFilter = file("$rootDir/checkstyle/spotbugs-exclude.xml")
+
```

Comment:
This changed a bit, but on my laptop Spotbugs is working.
I don't know how much was important the HTML report.

Reply:
The HTML report is important for human readability. So people can locate where the problem is.